src/main/kotlin/mate/academy/Ball.kt

```diff
@@ -0,0 +1,33 @@
+package mate.academy
+
+class Ball {
+    private var ballColor: String = ""
+    private var ballNumber: Int = 0
+
+    constructor()
+
```

Comment:
Then, if we delete default value, we don't need empty constructor, as we will init values during object creation